static/app/actionCreators/modal.tsx

```diff
@@ -388,3 +388,13 @@ export async function openInsightChartModal(options: InsightChartModalOptions) {
 
   openModal(deps => <Modal {...deps} {...options} />, {modalCss});
 }
+
+export async function openAddTempestCredentialsModal(options: {
+  organization: Organization;
+  project: Project;
+}) {
+  const mod = await import('sentry/components/modals/addTempestCredentialsModal');
+  const {default: Modal} = mod;
```

Comment:
I think this should work because you are not using `modalCss` 
```suggestion
  const Modal = await import('sentry/components/modals/addTempestCredentialsModal');
```

Reply:
Just tried it, it breaks everything if it is not destructured 😢